spec/support/request/shop_workflow.rb

```diff
@@ -16,9 +16,9 @@ def set_order(order)
     ApplicationController.any_instance.stub(:session).and_return({order_id: order.id, access_token: order.token})
   end
 
-  def add_product_to_cart
```

Comment:
I think that's correct - so let vars are acting kinda like global variables - not exactly transparent or flexible.